action.yml

```diff
@@ -8,6 +8,9 @@ inputs:
   generate-zip:
     description: 'Generate package zip file?'
     default: false
+  commit-message:
+    description: 'Use git last commit message if true, or "Update to version $VERSION from GitHub" if false else any text.'
```

Comment:
This should be matched with the corresponding flag in the read me file.

Reply:
Will create a commit with the changes